Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM golang:1.18-buster AS build
+
+WORKDIR /usr/lib
+RUN git clone https://github.com/Team-Kujira/core.git
```

Comment:
Shouldn't be necessary anymore since 7788ddebca69b4f537668c6a3be71e778c59c3db

Reply:
Awesome thanks @etienne-napoleone